### PR TITLE
Update Publizon URL + Refactor components to avoid re-renders

### PR DESCRIPTION
**Pull request by kasperbirch1**
#### Refactor components to avoid re-renders

This pull request includes the following changes:

- Made changes to `ModalReservationFormText`, `useAlternativeAvailableManifestation`, `ModalReservationFormText` and `handleSetImageLoaded` to improve performance.

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

### CODE REVIEW

File: src/components/cover/cover.tsx [hunk 1]
```diff
@@ -28,7 +28,9 @@ export const Cover = ({
   shadow
 }: CoverProps) => {
   const [imageLoaded, setImageLoaded] = useState<boolean | null>(null);
-  const handleSetImageLoaded = () => setImageLoaded(true);
+  const handleSetImageLoaded = useCallback(() => {
+    setImageLoaded(true);
+  }, []);
```
Comment (kasperg) [inline src/components/cover/cover.tsx:33]: I appreciate the intent here but I wonder if this callback should in fact not have any dependencies at all - such as the `id` or `idType` which will effect the source of the final image. If you think it is OK then move along :smile:

**Review by kasperg — CHANGES_REQUESTED** [review]
Without further reasoning I do not agree with the change to the publizon tokens.

Other changes are good - probably :smile:

File: src/components/cover/cover.tsx (hunk 1 above)
Comment (kasperbirch1) [reply src/components/cover/cover.tsx:33]: I am not sure if they should be there? 😅 If I put them in the dependency array I got this warning:
`React Hook useCallback has unnecessary dependencies: 'id' and 'idType'. Either exclude them or remove the dependency array.eslint`

**Review by kasperg — APPROVED** [review]
:+1: LGTM

**Review by jeppekroghitk — APPROVED** [review]

**Pull request merged**